databaseAnnotator/pom.xml

```diff
@@ -118,4 +118,14 @@
     </repository>
   </repositories>
 
+  <build>
+    <plugins>
+      <plugin>
+        <!-- required to build an executable jar -->
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
Building the jar might work without adding the plugin in this module (it's already in the parent). It doesn't hurt to add it though afaik, just makes it easier to configure at the module level in the future if we want. It was only really necessary for the `annotator` module b/c that one would no longer work with the spring boot upgrade

Reply:
if we're adding the plugin to all modules then maybe we should remove it from the parent?

Reply:
If we remove it from the parent then I guess we wouldnt need this to be skipped then in the `annotator` module, right?